content/posts/2024-07-30_crowdstrike-risk-at-the-edge.md

```diff
@@ -0,0 +1,39 @@
++++
+date = 2024-07-30T00:37:35+02:00
+title = "Risk assessment must happen at the sharp end"
+description = "I think the Crowdstrike is a great example of centralised risk management gone wrong."
+tags = ["security", "risk"]
+slug = "centralised-risk-management"
++++
+Today I wanted to write something about [the recent Crowdstrike Falcon debacle][crowdstrike-debacle] and about one contributing factor, namely _centralised risk management_.
+
+[crowdstrike-debacle]: https://en.wikipedia.org/wiki/2024_CrowdStrike_incident
+
+## The importance of local risk assessment
+
+I recently watched [the trailer][safety-differently-trailer] for the movie ["Safety Differently"][safety-differently] by Sidney Dekker. One of the things it talks about is that **the farther you are from a system, the worse your risk management controls (you come up with) will be**. Or to put it another way, the grassroots operators of a system are usually the best people to come up with controls and defend against catastrophes.
```

Comment:
What's in bold here is too definite. Need to losen up that language a bit. ChatGPT might help.